kas/config/common.yaml

```diff
@@ -15,6 +15,10 @@ local_conf_header:
     INIT_MANAGER="systemd"
     TEDGE_CONFIG_DIR = "/etc/tedge"
 
+    # customize tedge features
+    DISTRO_FEATURES:append = " tedge"
+    #TEDGE_FEATURES_ENABLE = "tedge-logs tedge-collectd tedge-docker tedge-p11-kit"
```

Comment:
We might need to use the append logic here as well, and explicit set the values and not rely on the layer defaults.

Reply:
See the comment above.